cni/network/plugin/main.go

```diff
@@ -266,7 +280,7 @@ func main() {
 		}
 	}
 
-	handled, err := handleIfCniUpdate(netPlugin.Update)
+	handled, _ := handleIfCniUpdate(netPlugin.Update)
```

Comment:
Should we log this error atleast ? if it is not nil

Reply:
its getting logged inside handleIfCniUpdate already